samples/java/testng-selenium/testng.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file necessary?

Reply:
@d-rede yes , we need this file to run the tests sequentially